### ISSUE TITLE
List of URLS

### ISSUE THREAD

**Issue by bootyhunt3r** [GerbenJavado/LinkFinder#92]
Hello man , 
Can you add a feature to like , add the js file from a text file , I have a file full of js file endpoints but I cannot pass it through linkfinder , if you can add this feature , it would be really helpful.

**Comment by velocity16902** [comment]
Yes please, even i'm trying to input list of js links saved in .txt file.

**Comment by bootyhunt3r** [comment]
Any update on this one ? Can you please update the script to work with the list of urls ?

**Comment by imon001** [comment]
you can try this simple bash script for use a list of .js urls in txt file. fst go to the directory where linkfinder is located.
then open terminal and type:
nano yourfile.sh
then copy-past the following lines:

#!/bin/bash
file=$1
lines=$(cat $file)
for line in $lines
do
python3 linkfinder.py -i $line -o cli
done

then save the file and exit with ctrl+x and enter.
then give execution permission to yourfile.sh with:
chmod +x ourfile.sh

now try it with:

./yourfile.sh js-list.txt

**Comment by bootyhunt3r** [comment]
@imon001 , thanks man , but it works fine for some js files and then it shows this error :
Error: invalid input defined or SSL error: <urlopen error [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:1123)>

**Comment by Sir-peck** [comment]
I was having issues with this, but your fix worked perfectly so just want to say thank you imon001

**Comment by barrett092** [comment]
Getting this error as well...
